files/en-us/web/css/box-sizing/index.html

```diff
@@ -30,7 +30,11 @@
 
 <ul>
  <li><code>content-box</code> gives you the default CSS box-sizing behavior. If you set an element's width to 100 pixels, then the element's content box will be 100 pixels wide, and the width of any border or padding will be added to the final rendered width, making the element wider than 100px.</li>
- <li><code>border-box</code> tells the browser to account for any border and padding in the values you specify for an element's width and height. If you set an element's width to 100 pixels, that 100 pixels will include any border or padding you added, and the content box will shrink to absorb that extra width. This typically makes it much easier to size elements.</li>
+ <li><code>border-box</code> tells the browser to account for any border and padding in the values you specify for an element's width and height. If you set an element's width to 100 pixels, that 100 pixels will include any border or padding you added, and the content box will shrink to absorb that extra width. This typically makes it much easier to size elements.
+  <div class="notecard note">
+   <p><strong>Note:</strong> Certain HTML elements, such as <code>&lt;button&gt;</code>, default to <code>border-box</code> behavior.</p>
```

Comment:
```suggestion
   <p><strong>Note:</strong> Certain HTML elements, <code>&lt;table&gt;</code> and Form controls default to <code>border-box</code> behavior. See <a href="https://html.spec.whatwg.org/multipage/rendering.html#tables-2">WHATWG HTML</a> for details on which is Form controls and have this behavior.</p>
```

Would this be a more explicit?

Reply:
I like this. Can we also move it outside of the `ul` to line 39?